Dockerfile.add-frontend-auth

```diff
@@ -0,0 +1,19 @@
+# base image
+FROM node:18.4
+# set working directory
+WORKDIR /usr/src/app
+# install and cache app dependencies
+COPY tornjak-frontend/package*.json ./
+ADD tornjak-frontend/package.json /usr/src/app/package.json
+RUN npm install
+# set env variables
+ARG REACT_APP_API_SERVER_URI
+ENV REACT_APP_API_SERVER_URI $REACT_APP_API_SERVER_URI
+ARG REACT_APP_AUTH_SERVER_URI
+ENV REACT_APP_AUTH_SERVER_URI $REACT_APP_AUTH_SERVER_URI
+# copy project files into the image
```

Comment:
should we also need a separate docker file for the case of no auth? I suppose we soon will make the frontend talk to the backend to get the relevant variables, so maybe it does not matter?

Reply:
It doesn't matter as you can decide not to use the variables: for example: if not provided